sdk/communication/azure-communication-identity/tests/test_communication_identity_client_async.py

```diff
@@ -6,42 +6,31 @@
 # --------------------------------------------------------------------------
 import pytest
 from datetime import timedelta
-from azure.core.credentials import AccessToken
 from azure.communication.identity.aio import CommunicationIdentityClient
 from azure.communication.identity import CommunicationTokenScope
 from azure.communication.identity._shared.utils import parse_connection_str
-from _shared.helper import URIIdentityReplacer, URIMsalUsernameReplacer
-from asynctestcase  import AsyncCommunicationIdentityTestCase
-from devtools_testutils import is_live
-from _shared.testcase import BodyReplacerProcessor
+from devtools_testutils import AzureRecordedTestCase, is_live
+from devtools_testutils.aio import recorded_by_proxy_async
 from _shared.communication_service_preparer import CommunicationPreparer
 from _shared.utils import get_http_logging_policy
-from utils import is_token_expiration_within_allowed_deviation
+from _shared.fake_token_credential import FakeTokenCredential
+from utils import is_token_expiration_within_allowed_deviation, generate_teams_user_aad_token, \
+    skip_get_token_for_teams_user_test
 from azure.identity.aio import DefaultAzureCredential
+from acs_identity_decorator import acs_identity_decorator
 
-class FakeTokenCredential(object):
-    def __init__(self):
-        self.token = AccessToken("Fake Token", 0)
-
-    async def get_token(self, *args):
-        return self.token
-class CommunicationIdentityClientTestAsync(AsyncCommunicationIdentityTestCase):
-    def setUp(self):
-        super(CommunicationIdentityClientTestAsync, self).setUp()
-        self.recording_processors.extend([
-            BodyReplacerProcessor(keys=["id", "token", "appId", "userId", "domain_name"]),
-            URIIdentityReplacer(),
-            URIMsalUsernameReplacer()])
-    
+class TestClientAsync(AzureRecordedTestCase):
```

Comment:
same comments that I left for sync tests

Reply:
Thanks! Updated this, too.